example/app.js

```diff
@@ -81,6 +81,10 @@ search.addWidget(
     templates: {
       header: '<div class="panel-heading">Price ranges</div>',
       item: require('./templates/and.html')
+    },
+    transformData: function(data) {
+      data.name = data.name.replace(/(\d+) - (\d+)/, '$$$1 - $$$2').replace(/> (\d+)/, '> $$$1');
```

Comment:
This is why I'd like to `deepClone`. We're just lucky here that the component doesn't use `facetValue` after calling `Template`.